presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -327,4 +335,29 @@ public static double infinity()
     {
         return Double.POSITIVE_INFINITY;
     }
+
+    @Description("convert a number to different bases (bin, hex, ...)")
+    @ScalarFunction
+    @SqlType(StandardTypes.VARCHAR)
+    public static Slice toBase(@SqlType(StandardTypes.BIGINT) long value, @SqlType(StandardTypes.BIGINT) long radix)
+    {
+        checkCondition(radix >= MIN_RADIX && radix <= MAX_RADIX,
+                INVALID_FUNCTION_ARGUMENT, "Radix must be in [%d, %d]: %d", MIN_RADIX, MAX_RADIX, radix);
```

Comment:
I used bracket notation and echoed the argument because that's how `ApproximateCountDistinctAggregations` and `AggregationNode` did it. I'll change this.